packages/quantic/force-app/main/default/lwc/quanticCategoryFacetValue/quanticCategoryFacetValue.js

```diff
@@ -16,10 +16,10 @@ export default class QuanticCategoryFacetValue extends LightningElement {
   }
   get categoryFacetLiClass() {
     if(this.activeparent) {
-      return "slds-var-m-horizontal_large slds-grid";
+      return "slds-var-m-left_large slds-grid";
     }
     if(this.nonactiveparent) {
```

Comment:
same return as default, so no need to put the condition

Reply:
lol so true